src/OpenTelemetry/Metrics/BaseExportingMetricReader.cs

```diff
@@ -91,7 +91,18 @@ internal override bool ProcessMetrics(in Batch<Metric> metrics, int timeoutMilli
             // TODO: Do we need to consider timeout here?
             try
             {
-                return this.exporter.Export(metrics) == ExportResult.Success;
+                OpenTelemetrySdkEventSource.Log.MetricReaderEvent("BaseExportingMetricReader calling Exporter.Export method.");
```

Comment:
would be nice if the name of the exporter is also part of the log message.

Reply:
Yeah, that would be good. Is there a way to do that without allocation using the existing event? Or do we have to create a new event which takes in another parameter?

Reply:
to do it without alloc, we'd likely need dedicated event (Or NonEvent which invokes the existing event)).

Reply:
Added the name of the exporter in the messages. I have the PR description to show the contents of the log file with the latest changes.